File: CHENGES/Python/31_01_2022/fibonacci.py
def fibonacciRec(num):
    if num <= 1:
        return num
    else:
        return fibonacciRec(num-1) + fibonacciRec(num-2)

def fibonacciLoop(num):
    n1, n2, i = 0, 1, 1

    if num <= 1:
        return num
    else:
        while i < num:
            n = n1 + n2
            n1 = n2
            n2 = n
            i += 1
        else:
            return n

print("*** Fibonacci ***")
num = int(input("Entry number of fib : "))
print(f"Fibonacci recursion : {fibonacciRec(num)}")
print(f"Fibonacci loop : {fibonacciLoop(num)}")

File: Arshak_Badalyan/c++/15_12_2021/fibonacci.cpp
#include <iostream>
 
int fib(int n) {
	if(n == 1 || n == 0) {
		return n;
	} else {
		return fib(n - 1) + fib(n - 2);
	}
}
int main () {
	int n;
	std::cout << "Please input the number of fibonacci sequence: "; 
	std::cin >> n;
    std::cout << "The " << n << " th fibonacci number is: " << fib(n) << std::endl;
    
	return 0;
}
